src/js/delete.js

```diff
@@ -16,37 +14,51 @@ function delete_relation(elem) {
   console.debug('Using globals: mei for element selection')
   // Assume no meta-edges for now, meaning we only have to
   // remove the SVG elem, the MEI node, and any involved arcs
-  var mei_id = get_id(elem)
-  var mei_he = get_by_id(mei, mei_id)
-  var svg_hes = []
-  var metarel = get_class_from_classlist(elem) == 'metarelation'
-  var mei_graph = getMeiGraph()
-  var draw_contexts = getDrawContexts()
+  const mei_id = get_id(elem)
+  const mei_he = get_by_id(mei, mei_id)
+  const svg_hes = []
+  const is_meta_relation = get_class_from_classlist(elem) == 'metarelation'
+  const mei_graph = getMeiGraph()
+  const draw_contexts = getDrawContexts()
   for (const draw_context of draw_contexts) {
-    let svg_he = get_by_id(document, draw_context.id_prefix + mei_id)
+    const svg_he = get_by_id(document, draw_context.id_prefix + mei_id)
     if (svg_he) {
       svg_hes.push(svg_he)
-      if (!metarel)
-        unmark_secondaries(draw_context, mei_graph, mei_he)
+      if (!is_meta_relation) unmark_secondaries(draw_context, mei_graph, mei_he)
     }
   }
 
-  var arcs =
-    Array.from(mei.getElementsByTagName('arc')).filter((arc) => {
-      return arc.getAttribute('to') == '#' + elem.id ||
-              arc.getAttribute('from') == '#' + elem.id
-    })
-  var removed = arcs.concat(svg_hes)
+  // Find all arcs related to this element
+  const arcs = Array.from(mei.getElementsByTagName('arc')).filter(arc => {
+    return arc.getAttribute('to') == '#' + elem.id || arc.getAttribute('from') == '#' + elem.id
+  })
```

Comment:
Are these three lines actually different (apart from formatting)? If not, please revert to original formatting for history purposes.